src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -155,72 +199,216 @@ Add in `routingOperations` array following definition:
   }
 ```
 
+Here we provide not one, but three different handlers! Some of them even have some additional configurations defined explicitly or in another file. You'll learn about `fragments` and their power and functionalities in a different tutorial.
```

Comment:
1. What does it mean to have more than one handler? Make a note that they are called in the order they're defined, processing the same request.
2. Make a note that handler configurations are passed as `JsonObject`s in java. It's not obvious what reads the configuration
3. If someone don't know what fragments are they will be confused why you bring them up. Maybe write a short note that they are used for communication between handlers.

Reply:
done